version/version.go

```diff
@@ -25,7 +25,9 @@ import (
 )
 
 var (
-	Version = "2.1.0-alpha.0+git"
+	// MinVersion of etcd this version can work with
+	MinVersion = "2.0.0"
```

Comment:
I thought we decided we were only using the major and minor not the patchlevel for the cluster version.